Makefile

```
@@ -27,6 +27,7 @@ STAGE1_STACKER ?= ./stacker-dynamic
 STACKER_DEPS = $(GO_SRC) go.mod go.sum
 
 stacker: $(STAGE1_STACKER) $(STACKER_DEPS) cmd/stacker/lxc-wrapper/lxc-wrapper.c
+	echo "$(STACKER_BUILD_BASE_IMAGE)"
```

```suggestion
	@echo "$(STACKER_BUILD_BASE_IMAGE)"
```

Adding the @ tells Make not to print out the command name before the output